QUICK_START.md

```diff
@@ -0,0 +1,19 @@
+# Rubrik SharePoint Metrics Collector
+
+## Overview
+This repo contains the PowerShell script to get statistics on O365 environment.
+
+Required Modules:
+```
+Install-Module -Name Microsoft.Online.SharePoint.PowerShell -RequiredVersion 16.0.8029.0
+Install-Module -Name SharePointPnPPowerShellOnline -RequiredVersion 3.21.2005.1
+```
+
+## Using ths module
+$userid = "admine@domain.com"
+$password = "MyPass123!"
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High <!-- 9d1e62a1-06f6-4253-8112-f6d2fd9479d5 -->